atsign-foundation/at_app_cli
# Bump version to 4.1.0

**- What I did**

Made the appropriate updates in each CHANGELOG.md, pubspec.yaml, and version.dart in at_app.
Fixed some issues with publishing.

**- How I did it**

Local changes.

**- How to verify it**

See the file changes, try at_app 4.1.0-rc+1.

```flutter pub global activate at_app 4.1.0-rc+1```

**- Description for the changelog**

Bump version to 4.1.0
